src/definitions/observableDefinitions.js

```diff
@@ -237,5 +237,29 @@ export const OBSERVABLE_TYPES = [
       { field: "description", type: "longtext", label: "Description", displayList: false },
       { field: "created", type: "datetime", label: "Created", displayList: true }
     ]
+  },
+  {
+    name: "User-Account",
```

Comment:
```suggestion
    name: "User Account",
```

To be inline with other observable names (except user-agent but that's a special case)